test/test_common/utility.cc

```diff
@@ -238,6 +238,23 @@ AssertionResult TestUtility::waitForGaugeEq(Stats::Store& store, const std::stri
   return AssertionSuccess();
 }
 
+AssertionResult TestUtility::waitForGaugeDestroyed(Stats::Store& store, const std::string& name,
+                                                   Event::TestTimeSystem& time_system,
+                                                   std::chrono::milliseconds timeout) {
```

Comment:
suggest dropping the timeout from this API and others since you are not specifying it anywhere.

Reply:
fixed